pkg/filter/driftignore.go

```diff
@@ -107,6 +107,18 @@ func (r *DriftIgnore) IsTypeIgnored(ty resource.ResourceType) bool {
 		return false
 	}
 
+	return r.shouldIgnoreType(ty)
+}
+
+func (r *DriftIgnore) shouldIgnoreType(ty resource.ResourceType) bool {
+	for _, pattern := range r.ignorePatterns {
+		// If a line start with a `!` and if the type match, we should not ignore it
+		if strings.HasPrefix(pattern, "!") &&
+			strings.HasPrefix(pattern[1:], fmt.Sprintf("%s.", ty)) {
+			return false
+		}
```

Comment:
Why not just do `if strings.HasPrefix(pattern, fmt.Sprintf("!%s.", ty)) {` here ? May be I'm missing something

Reply:
True, let me fix that 👍🏻